news/_posts/releases/2019-11-28-prestashop-1-7-6-2-maintenance-release.md

```diff
@@ -0,0 +1,62 @@
+- One bug impacting contributor and partner payment modules due to amount paid no longer validated when creating an order ([#15834](https://github.com/PrestaShop/PrestaShop/issues/15834))
+- Three bugs related to the CLDR implementation in the 1.7.6.0 which were provoking errors in both suppliers and orders pages after deleting a currency ([#15376](https://github.com/PrestaShop/PrestaShop/issues/15376)), ([#15486](https://github.com/PrestaShop/PrestaShop/issues/15486)) and ([#15956](https://github.com/PrestaShop/PrestaShop/issues/15956))
+- A few regressions related to the migration of the Customer page:
+- Customer creation with SIRET field ([#16167](https://github.com/PrestaShop/PrestaShop/issues/16167))
+- Customer listing page bugs on links, filters and export features ([#15831](https://github.com/PrestaShop/PrestaShop/issues/15831)), ([#10293](https://github.com/PrestaShop/PrestaShop/issues/10293)) and ([#15875](https://github.com/PrestaShop/PrestaShop/issues/15875)) 
+- Customers in a multi-store environment ([#16043](https://github.com/PrestaShop/PrestaShop/issues/16043))
+- Error on creating new Brand Address when VAT identification number is mandatory ([#16121](https://github.com/PrestaShop/PrestaShop/issues/16121))
+- Quantity discounts not shown for default variant ([#16417](https://github.com/PrestaShop/PrestaShop/issues/#16417))
+- Revert back to using product URL (including combination) rather than canonical one in the miniatures (SEO) ([#14765](https://github.com/PrestaShop/PrestaShop/issues/14765))
+- Fixed PrestaShop Command Line Interface (CLI) cache clearing capabilities ([#15324](https://github.com/PrestaShop/PrestaShop/issues/15324))
+
+Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.6.2).
+
+
+## Important changes
+By Allowing CLDR to display deleted/inactive currencies [#15643](https://github.com/PrestaShop/PrestaShop/pull/15643) we had to add new methods to interfaces `DataSourceInterface`, `CurrencyDataProviderInterface` and `RepositoryInterface` to be able to fetch currencies regardless of them being deleted or inactive. This requires to update any class that implemented that interface.
+
+We had to revert the fix we made for the `object_lang` table issue in 1.7.6.0 regarding multi-store data [#11876](https://github.com/PrestaShop/PrestaShop/pull/11876) as it introduced a regression [#15274](https://github.com/PrestaShop/PrestaShop/issues/15274) that changed how multistore data was persisted. More details here [#15621](https://github.com/PrestaShop/PrestaShop/pull/15621).
+
+
+## Acknowledgments
+
+Contributors to this patch version, from both the Core team and the community at large: 
+Aurélien Pelletier, Jean-François Viguier, Jonathan Lelievre, Khouloud Belguith, Mathieu Ferment, Matthieu Rolland, Pablo Borowicz, Pierre Rambaud, Franck Lefevre, Thomas Baccelli.
```

Comment:
Only us participated to 1.7.6.2 😮 no external contributor ?

Reply:
Can we add 202-ecommerce ? PR #16258 was actually a backport of https://github.com/PrestaShop/PrestaShop/pull/16072 so initial credit should go to them